src/app/components/dynamic-list/DynamicListItem.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import PropTypes from "prop-types";
+import PersonInShield from "../../icons/PersonInShield";
+import TickInShield from "../../icons/TickInShield";
+import Person from "../../icons/Person";
+
+const propTypes = {
+    title: PropTypes.string,
+    desc: PropTypes.string,
+    icon: PropTypes.oneOf(["Cross", "Person", "PersonInShield", "TickInShield", "Warning"]),
+    iconColor: PropTypes.oneOf(["standard", "blue", "green", "red"]),
+    iconName: PropTypes.string,
+    buttonName: PropTypes.string,
+    buttonCallback: PropTypes.func,
+};
+
+const DynamicListItem = props => {
+    const colorBlack = "#000000";
+    let icon;
+    switch (props.icon) {
+        case "Shield-person":
+            icon = <PersonInShield classes="dynamic-list-item__icon" viewBox="0 0 15 15" fillColor={colorBlack} ariaLabel={props.iconName} />;
```

Comment:
Maybe consider extracting the `viewBox` property into its own variable, similar to what you've done with `colorBlack`. On the off chance we ever need to update it, we could do so in one place rather than three